dali/base/dadfs.cpp

```diff
@@ -4704,6 +4705,20 @@ protected: friend class CDistributedFilePart;
         else
             return false;
     }
+    virtual double getCost(const char * cluster) override
+    {
+        VStringBuffer path("storage/planes[@name='%s']", cluster);
+        Owned<IPropertyTree> plane = queryGlobalConfig().getPropTree(path);
+        if (!plane)
+            return 0.0;
+        double storageCostDaily = plane->getPropReal("cost/@storageAtRest", 0.0) * 12 / 365;
```

Comment:
so /storageAtRest' is supposed to represent a months worth of cost?
Why has this unit been chosen? (because Azure uses it?).

Wouldn't per day (or even per hour) be a better flexible unit for us ?
Then we'd scale whatever units the provider chose to use, into our standard.